src/index.html

```diff
@@ -7,6 +7,289 @@
+            <span class="service-card__title">Electric</span>
+          </div>
+
+        </div>
+      </section>
+<!-- ------------------ EXPERIENCE --------------------- -->
+      <section class="experience">
+        <div class="experience__cards">
+
+          <div class="experience-card card-right">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/clients.svg" alt="clients icon">
+            </div>
+            <span class="experience_card__digit">84</span>
+            <span class="experience-card__description">Happy Clients</span>
+          </div>
+
+          <div class="experience-card card-left">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/projects.svg" alt="projects icon">
+            </div>
+            <span class="experience_card__digit">123</span>
+            <span class="experience-card__description">Projects Completed</span>
+          </div>
+
+          <div class="experience-card card-right">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/awards.svg" alt="awards icon">
+            </div>
+            <span class="experience_card__digit">37</span>
+            <span class="experience-card__description">Awards Win</span>
+          </div>
+
+          <div class="experience-card card-bottom">
+            <div class="experience-card__icon">
+              <img src="./assets/icons/business.svg" alt="business icon">
+            </div>
+            <span class="experience_card__digit">30</span>
+            <span class="experience-card__description">Years in Business</span>
+          </div>
+
+        </div>
+        <div class="experience__description">
+          <span class="experience__description__title">30 Years Experience</span>
+          <p class="experience__description__content">Our company has been the leading provided construction services to clients throughout the USA since 1988.</p>
+          <button href="#" class="experience__description__btn">Contact Us</button>
+        </div>
+      </section>
+<!-- ------------------ CONSULTATION --------------------- -->
+      <section class="consultation">
+        <img src="./assets/images/room-background.png" alt="room background photo" class="sonsultation__img">
+        <div class="consultation__info">
+          <span class="consultation__info-descr">Free consultation with exceptional quality</span>
+          <span class="consultation__info-tel">Just one call away: <a href="tel:+84 1102 2703">+84 1102 2703</a></span>
+        </div>
+        <button href="#" class="consultation__btn">Get your consultation</button>
+      </section>
+<!-- ------------------ PROJECTS --------------------- -->
+      <section class="projects">
+        <h2 class="projects__title">Projects</h2>
+        <div class="projects__tabs-body">
+          <div class="projects__tabs">
+            <a href="#" class="projects-tab">All</a>
```

Comment:
Incorrect naming for the element.
According to your class, it is either a separate block or a descendent element, but both are invalid in this case